.env

```diff
@@ -0,0 +1 @@
+ MONGO_URI = mongodb://northUser:bGJucFTFliN8qX2Z@ac-gkbjsml-shard-00-00.oxafelt.mongodb.net:27017,ac-gkbjsml-shard-00-01.oxafelt.mongodb.net:27017,ac-gkbjsml-shard-00-02.oxafelt.mongodb.net:27017/?ssl=true&replicaSet=atlas-yrtq9t-shard-0&authSource=admin&retryWrites=true&w=majority
```

Comment:
You should never check files with this type of sensitive data into git. You can read about how Next.js recommends that you use environment variables here: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

In summary, you can have a `.env.local` file that will automatically be consumed by Next.js, and it should be ignored by git so that it's never included (add it to `.gitignore`).